spyderlib/plugins/editor.py

```diff
@@ -1521,7 +1521,7 @@ def _clone_file_everywhere(self, finfo):
             editor = editorstack.clone_editor_from(finfo, set_current=False)
             self.register_widget_shortcuts("Editor", editor)
     
-    @Slot()
+#    @Slot(str)

```

Comment:
This is an error. Please leave this as

```python
@Slot()
@Slot(str)
def new(self, fname=None, editorstack=None, text=None):
```

We need those two slot signatures for the `new` method to work correctly.